data/labwc-portals.conf

```diff
@@ -0,0 +1,2 @@
+[preferred]
+default=wlr;gtk;
```

Comment:
What if a user only has the kde portals installed? Wouldn't this completely break all portals for that user?
It seems there is a `*` default as well: https://github.com/flatpak/xdg-desktop-portal/blob/main/doc/portals.conf.rst.in#L117

So I assume a better default would be `default=wlr;*` here? (recognizing that labwc is not a desktop environment and does not prefer one portal implementation over another, at least outside of the -wlr portal interfaces).

Reply:
I thought about this and chose poorly. Apologies.